collectors/auth0/auth0_collector.js

```diff
@@ -84,7 +84,8 @@ class Auth0Collector extends PawsCollector {
             priority: 11,
             progName: 'Auth0Collector',
             message: JSON.stringify(msg),
-            messageType: 'json/auth0'
+            messageType: 'json/auth0',
+            application_id: process.env.application_id
```

Comment:
I think we need to get away from collectors using `process.env` directly. Let's use base class properties, like
```
let collector = this;
...
application_id: collector.application_id
```